alf/algorithms/actor_critic_algorithm.py

```diff
@@ -151,7 +151,8 @@ def rollout_step(self, inputs: TimeStep, state: ActorCriticState):
         action_distribution, actor_state = self._actor_network(
             inputs.observation, state=state.actor)
 
-        action = dist_utils.sample_action_distribution(action_distribution)
+        action, log_prob = dist_utils.sample_action_distribution(
```

Comment:
A documentation briefly explaining why the corresponding `log_prob` is stored can help the future the reader.

Reply:
> A documentation briefly explaining why the corresponding `log_prob` is stored can help the future the reader.

Good point. Added more documentation in the sampling function. Also added related unit tests.